packages/medusa/src/api/routes/admin/customers/list-customers.ts

```diff
@@ -28,26 +27,9 @@ export default async (req, res) => {
 
   const customerService: CustomerService = req.scope.resolve("customerService")
 
-  const selector: AdminListCustomerSelector = {}
-
-  if (validated.q) {
-    selector.q = validated.q
-  }
-
-  let expandFields: string[] = []
-  if (validated.expand) {
-    expandFields = validated.expand.split(",")
-  }
-
-  const listConfig: FindConfig<Customer> = {
-    relations: expandFields,
-    skip: validated.offset,
-    take: validated.limit,
-  }
-
-  const [customers, count] = await customerService.listAndCount(
-    selector,
-    listConfig
+  const [customers, count] = await customerController.listAndCount(
+    customerService,
+    validated
```

Comment:
Would it be nice to rename that variable as validatedQueryParams in order to be more understandable?